config.py

```diff
@@ -32,7 +40,6 @@ class TestingConfig(Config):
     Testing configurations
     """
 
-    SECRET_KEY = 'some value'
     TESTING = True
```

Comment:
We taked about removing `TESTING` for Production environment

Reply:
I set `TESTING = False` in `class Config` for all environments.
I will add it for production as well.

Reply:
Sorry about that. I thought it was `ProductionConfig`, just realized it's `TestingConfig`